src/cluster/distribution.rst

```diff
@@ -0,0 +1,111 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _cluster/distribution:
+
+==============================
+Custom Erlang Network Protocol
```

Comment:
Not sure the network protocol itself would be custom? It's either TCP or TLS. Perhaps we can re-word to indicate that this is specifically to allow using TLS for Erlang distribution between nodes, with the ability to connect to some nodes using TCP as well.

Reply:
How about `TLS Erlang Distribution`?